random_joke.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from classes import JokeApi
+
+app = Flask(__name__)
+
+@app.route("/random-joke")
+def return_joke():
+
+    joke = JokeApi.get_random_joke()
+    
+    return f"{joke['setup']}...{joke['punchline']}"
```

Comment:
See how clean this is now? This is the beauty of having logic for various parts of your application live in their own domains. 

Now, imagine you wanted to have a separate endpoint that returned 5 jokes (not a great example but serves the purpose) then you can simply reuse the `JokeApi` class rather than re-writing the API implementation in another function.

Reply:
It is so much cleaner   😊. It's starting to make much more sense! Thanks to you ;)